### PR TITLE
groupManager, workManager, and WorkManagerController done

### PR DESCRIPTION
**Pull request by Tseateen**

### CODE REVIEW

File: src/main/InterfaceAdapter/WorkManagerController.java [hunk 1]
```diff
@@ -1,26 +1,82 @@
...
-import main.UsesCases.IWorkManager;
-import main.UsesCases.WorkList;
+import main.UsesCases.*;
+
+import java.util.ArrayList;
 
 public class WorkManagerController {
 
-    private final WorkList workList;
+    private final IWorkList workList;
```
Comment (ChiehAnChang) [inline src/main/InterfaceAdapter/WorkManagerController.java:9]: Don't delegate DataList

File: src/main/InterfaceAdapter/FacadeSys.java [hunk 2]
```diff
@@ -188,7 +188,7 @@ public void extendWork(String days, String work_id) {
 
     public boolean checkWorkExist(String work_id) {
         return this.workFacade.workExist(work_id);
-    }
+    }//D
```
Comment (ChiehAnChang) [inline src/main/InterfaceAdapter/FacadeSys.java:191]: Accidentally mistake?

File: src/main/UsesCases/WorkManager.java [hunk 3]
```diff
@@ -55,7 +67,6 @@ public void autoChangeState(Workable work) {
         work.setState("Pending");
 
     }
-
     /**
      * Change the Work information with different options.
      *
```
Comment (ChiehAnChang) [inline src/main/UsesCases/WorkManager.java:72]: Don't decide which option inside use case

File: src/main/UsesCases/IGroupManager.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+public interface IGroupManager {
+    public void addMembers(List<String> list, Group group);
+    public void deleteMember(String userID, IGroupList groupList);
+    public void resetMember(Group group);
+    public void changeLeader(Group group, String user);
+    public void resetGroup(Group group);
+    public boolean Distributor(String workID, String userID, IGroupList groupList);
+    public boolean isMember (String userID, String workID, IGroupList groupList);
+    public boolean verifierLeader(String userID, String workID, IGroupList groupList);
```
Comment (ChiehAnChang) [inline src/main/UsesCases/IGroupManager.java:15]: Is it necessary to put verifierLeader into GroupManager?

Comment (Lily-alive) [reply src/main/UsesCases/IGroupManager.java:15]: If verifierLeader is not necessary, then isMember is not necessary as well:(

**Pull request merged**